src/bentoml/bentos.py

```diff
@@ -349,21 +353,40 @@ def build(
         conda=conda,
     )
 
-    return Bento.create(
-        build_config=build_config,
-        version=version,
-        build_ctx=build_ctx,
-    ).save(_bento_store)
+    build_args = ["bentoml", "build"]
+
+    if build_ctx is None:
+        build_ctx = "."
+    build_args.append(build_ctx)
+
+    if version is not None:
+        build_args.extend(["--version", version])
+
+    with tempfile.NamedTemporaryFile(
+        "w", encoding="utf-8", prefix="bentoml-build-", suffix=".yaml"
+    ) as f:
+        build_config.to_yaml(f)
+        bentofile_path = os.path.join(os.path.dirname(f.name), f.name)
+        build_args.extend(["--bentofile", bentofile_path])
+        try:
+            output = subprocess.check_output(build_args)
+        except subprocess.CalledProcessError as e:
+            logger.error("Failed to build BentoService bundle: %s", e)
+            raise
+
+    rgx = re.compile(r"^Successfully built.*")
+    tag_parse = [s for s in output.decode("utf-8").split("\n") if rgx.match(s)][0]
+    return get(tag_parse.split(" ")[-1])
```

Comment:
Maybe add a `-s` / `--short` option to `bentoml build` so only the tag name is returned so we won't have to parse with regex here.